streamwebs_frontend/streamwebs/templates/streamwebs/datasheets/camera_point_add.html

```diff
@@ -66,9 +66,6 @@
 
 <div class="container">
     {% if user.is_authenticated %}
```

Comment:
Hi. Deleting the success message is good because there is already a flash message in the ``add_camera_point`` view. You can also delete the ``if user.is_authenticated`` check and ``You must be logged in ... ``since the view is already login-protected.